os/se/ameba/security_ameba_wrapper.c

```diff
@@ -1474,6 +1494,10 @@ int ameba_hal_delete_storage(uint32_t ss_idx)
 static struct sec_ops_s g_ameba_ops = {
 	ameba_hal_init,
 	ameba_hal_deinit,
+#ifdef CONFIG_ARM_TRUSTZONE
```

Comment:
please consider another sec driver. (ex. konai.c)

Reply:
can you explain more? this modification is valid only for the board which has trustzone support in TizenRT. Currently it is only Realtek amebad, so I added in ameba wrapper

Reply:
if CONFIG_ARM_TRUSTZONE enabled and select konai solution, build error will be happen.
If konal can't apply TRUSTZONE, below is needed.
```
#ifdef CONFIG_ARM_TRUSTZONE
     null,
     null,
#endif
```

Reply:
If CONFIG_ARM_TRUSTZONE is enabled, it means SE will be implemented in TrustZone secure image
It is mandatory to implement allocate/free then, If it is not present, none of the features of SE can be used  otherwise TrustZone based SE cannot be used.

Reply:
Just enable TRUSTZONE and select another se module then build.
You can see build error.

Reply:
You already added hal_allocate_context & hal_free_context as a basic sec operation.
But you override only here.
Let's enable TRUSTZONE & select other security module & build.
And see what happened.